src/_css/AllocationsComponent.scss

```diff
@@ -12,3 +12,9 @@
   position: absolute;
   width: 90px;
 }
+
+@media (max-width: 767px) {
```

Comment:
Make the max width be `768px`. Same comment applies in all areas you have used `767px`

Reply:
Instead of changing the width, I have added a media query to cater for it